fdbserver/Knobs.cpp

```diff
@@ -182,6 +182,8 @@ ServerKnobs::ServerKnobs(bool randomize, ClientKnobs* clientKnobs) {
 	init( DEBOUNCE_RECRUITING_DELAY,                             5.0 );
 	init( DD_FAILURE_TIME,                                       1.0 ); if( randomize && BUGGIFY ) DD_FAILURE_TIME = 10.0;
 	init( DD_ZERO_HEALTHY_TEAM_DELAY,                            1.0 );
+	init( REBALANCE_MAX_RETRIES,                                 100 );
+	init( DD_OVERLAP_PENALTY,                                  10000 );
```

Comment:
It will be better to add a comment:
`DD_OVERLAP_PENALTY` should be larger than the max possible team number a server may have. 

Non-blocking though.